src/main/webapp/app/pages/ActionableGenesPage.tsx

```diff
@@ -549,28 +577,181 @@ export default class ActionableGenesPage extends React.Component<
         >
           <Then>
             <Row
```

Comment:
Considering you already put the CollapseStatus together, maybe wrap all view components within a for loop instead of duplicating the same component?